plugins/org.eclipse.fordiac.ide.debug.ui/src/org/eclipse/fordiac/ide/debug/ui/view/FBDebugViewEditPartFactory.java

```diff
@@ -57,13 +56,10 @@ protected EditPart getPartForElement(final EditPart context, final Object modelE
 		if (modelElement instanceof InterfaceValueEntity) {
 			return new InterfaceValueEditPart();
 		}
-		if (modelElement instanceof EventValueEntity) {
-			return new EventValueEditPart();
-		}
-		if (modelElement instanceof Event && ((Event) modelElement).isIsInput()) {
-			// for input events we would like to use a button
-			return new DebugInputEventEditPart();
```

Comment:
do you still need this class or can we delete it completely?

Reply:
thx for noting. Should be removed.